doc/languages.rst

```diff
@@ -549,6 +549,10 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       Check syntax and lint with `ESLint <http://eslint.org/>`_.
 
+      .. note::
+
+         This syntax checker requires the presence of a configuration file.
+
```

Comment:
I guess that's for the other pull request as well, right?  Looks as if it's related to the eslint change.